### ISSUE TITLE
#review

### ISSUE THREAD

**Issue by MarcosJoaquinDev** [Micaela777/desafio-final-modulo-4#1]
Hola mica quedo excelente el desafío, me encanto el diseño y el modo responsive no tiene problemas, el único de detalle que encontré fue en la parte de /servicios si le agregas elementos al contenedor "services-to-give-container " el orden se rompe

**Comment by Micaela777** [comment]
@MarcosJoaquinDev Hola Marcos! Gracias por tu review, en cuánto al detalle, te referís a que el orden sigue lineal y a medida que se le agregan elementos no se van acomodando abajo?

**Comment by MarcosJoaquinDev** [comment]
claro, se alinean solo  horizontal, le podes poner un " flex-wrap : wrap":  hace que se acomode solo y manda los elementos para abajo en caso de que no entren

**Comment by Micaela777** [comment]
@MarcosJoaquinDev Genial! Muchas gracias por la corrección 🤗
